### ISSUE TITLE
Consider transferring this repo to coq-community

### ISSUE THREAD

**Issue by palmskog** [ybertot/canonical_forms#1]
Since CoqEAL is set to be transferred to coq-community soon (https://github.com/coq-community/manifesto/issues/127), please consider proposing a transfer of this repository as well ([instructions](https://github.com/coq-community/manifesto/blob/master/CONTRIBUTING.md#proposing-a-new-package)). Recall that you can still maintain the repo as you see fit, but with the added bonus that people including Théo and myself can fine-tune continuous integration to test against different versions of Coq and MathComp. 

Moreover, coq-community makes it easier to relocate content between repositories to their "proper" place, so that code receives more maintainer care. See for example the [topology](https://github.com/coq-community/topology) repo which recently absorbed its dependency zorns-lemma.

**Comment by ybertot** [comment]
I am all in favor of transfering this to coq-community.  I don't know how much work is involved in this.  I expect that I will at least have to devote a few hours understanding the various templates and how to fill them up.   I will try to work on it before the end of September.

**Comment by palmskog** [comment]
There is no requirement to have metadata based on templates (although it helps, in the long term). The only form you need to fill in is for the [new issue](https://github.com/coq-community/manifesto/issues/new?labels=move-project&template=1-move-a-project.md&title=Proposal+to+move+project+X+to+coq-community) to propose the transfer. Community members including myself can help with the rest in due time once the transfer is done.
